tests/generative/test_query_model.py

```diff
@@ -1,14 +1,18 @@
+import itertools
 import os
 
 import hypothesis as hyp
 import hypothesis.strategies as st
 import pytest
+import sqlalchemy.exc
 
 from databuilder.query_model.nodes import Column, TableSchema, count_nodes, node_types
 
 from ..conftest import QUERY_ENGINE_NAMES, engine_factory
 from . import data_setup, data_strategies, variable_strategies
 
+ignore_result = object()
```

Comment:
ronto-nit: I've generally seen sentinels written in all caps as constants.

Reply:
Oh, I'm very happy to do that. I started with all caps and changed it because I thought it was my Java heritage showing through.